docs/guide/advanced/teleport.md

```diff
@@ -187,6 +187,22 @@ test('teleport', async () => {
 })
 ```
 
+You have a possibility to stub teleport without creating any external HTML 
```

Comment:
Maybe just: 
```
You can stub teleport by using `teleport: true`:
```

Would you mind updating the docs in french as well please? You can use: 

```
Vous pouvez remplacer teleport par un stub en utilisant `teleport: true`&nbsp;:
```

Reply:
Sure! Applied changes